pages/common/htop.md

```diff
@@ -0,0 +1,16 @@
+# htop
+
+> An interactive process viewer for Unix.
+> More Information: <https://hisham.hm/htop/>
```

Comment:
I would maybe use this link: https://linux.die.net/man/1/htop (more general one)